diff -urN icmpecho/Makefile icmpecho.fixed/Makefile
--- icmpecho/Makefile	2012-02-01 00:16:15.000000000 +0000
+++ icmpecho.fixed/Makefile	2013-01-28 18:58:57.292865005 +0000
@@ -1,7 +1,7 @@
 PROJECT = pyraw
 OUTPUT  = $(DESTDIR)$(ZENHOME)/bin/$(PROJECT)
-CFLAGS  = -I$(DESTDIR)$(ZENHOME)/include/python2.7
-LDFLAGS = -L$(DESTDIR)$(ZENHOME)/lib
+CFLAGS  = -I$(DESTDIR)$(ZENHOME)$(VIRTUAL_ENV)/include/python2.7
+LDFLAGS = -L$(DESTDIR)$(ZENHOME)$(VIRTUAL_ENV)/lib
 LDLIBS  = -lpython2.7 -ldl
 CC     ?= gcc
 
@@ -9,7 +9,7 @@
 	LDFLAGS += -framework CoreFoundation -u _PyMac_Error
 else
 	LDLIBS += -lpthread -lutil -lm
-	LDFLAGS += -Xlinker -export-dynamic -Wl,-rpath,$(ZENHOME)/lib
+	LDFLAGS += -Xlinker -export-dynamic -Wl,-rpath,$(DESTDIR)$(ZENHOME)/lib
 endif
 
 all: $(OUTPUT)
